.arm.docker-env.sample

```diff
@@ -0,0 +1,2 @@
+DOCKER_HOST="tcp://192.168.2.2:2375"
```

Comment:
Agreed.  I don't think we have instructions anywhere on how to actually build arm.